app/mailers/request_mailer.rb

```diff
@@ -345,15 +392,37 @@ def self.alert_new_response_reminders_internal(days_since, type_code)
   # Send email alerts for requests which need clarification. Goes out 3 days
   # after last update of event.
   def self.alert_not_clarified_request
-    info_requests = InfoRequest.find(:all, :conditions => [ "awaiting_description = ? and described_state = 'waiting_clarification' and info_requests.updated_at < ?", false, Time.now - 3.days ], :include => [ :user ], :order => "info_requests.id" )
+    info_requests = InfoRequest.
+                      find(:all,
+                           :conditions =>
+                            [
+                              "awaiting_description = ?
+                               AND described_state = 'waiting_clarification'
+                               AND info_requests.updated_at < ?",
+                              false,
+                              Time.now - 3.days
```

Comment:
Do not use `Time.now` without zone. Use one of `Time.zone.now`, `Time.now.current`, `Time.now.in_time_zone`, `Time.now.utc`, `Time.now.getlocal`, `Time.now.iso8601`, `Time.now.jisx0301`, `Time.now.rfc3339`, `Time.now.to_i`, `Time.now.to_f` instead.